packages/synapse-react-client/src/utils/functions/TableColumnSchemaUtils.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ColumnChange,
+  ColumnModel,
+  TableUpdateTransactionRequest,
+} from '@sage-bionetworks/synapse-types'
+import { SetOptional } from 'type-fest'
+import { isEqual } from 'lodash-es'
+import { createColumnModels } from '../../synapse-client'
+
+/**
+ * Creates the column models for a change request and returns
+ * the change request to be passed to the Synapse backend.
+ * @param accessToken
+ * @param tableId
+ * @param oldSchema
+ * @param proposedSchema
+ */
+export async function createTableUpdateTransactionRequest(
```

Comment:
Moved without changes from TableColumnSchemaFormUtils.

I moved this into `src/utils` because I wanted to refactor to call this from inside the custom `useUpdateTable` mutation, but I didn't want the mutation to import from the `src/components` folder.